internal/iam/repository_role.go

```diff
@@ -182,13 +182,13 @@ func (r *Repository) DeleteRole(ctx context.Context, withPublicId string, _ ...O
 }
 
 // ListRoles in a scope and supports WithLimit option.
-func (r *Repository) ListRoles(ctx context.Context, withScopeId string, opt ...Option) ([]*Role, error) {
+func (r *Repository) ListRoles(ctx context.Context, withScopeIds []string, opt ...Option) ([]*Role, error) {
```

Comment:
Can you update the godoc for the method to reflect the changes in the signature?

Reply:
Sure, done.